framework/benchmark/process.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+const testing = require('../dist-node/testing');
+const os = require('os');
+const { address, ed, legacy, bls } = require('@liskhq/lisk-cryptography');
+const { Mnemonic } = require('@liskhq/lisk-passphrase');
+const { codec } = require('@liskhq/lisk-codec');
+const { Transaction, TAG_TRANSACTION } = require('@liskhq/lisk-chain');
+const { getGeneratorPrivateKeyFromDefaultConfig } = require('../dist-node/testing/fixtures');
+const { transferParamsSchema } = require('../dist-node/modules/token/schemas');
+
+const defaultTokenID = chainID => Buffer.concat([chainID, Buffer.from([0, 0, 0, 0])]);
+const createAccount = () => {
+	const passphrase = Mnemonic.generateMnemonic(256);
+	const keys = legacy.getPrivateAndPublicKeyFromPassphrase(passphrase);
+	const blsPrivateKey = bls.generatePrivateKey(Buffer.from(passphrase, 'utf-8'));
+	const blsPublicKey = bls.getPublicKeyFromPrivateKey(blsPrivateKey);
+	const blsPoP = bls.popProve(blsPrivateKey);
+	return {
+		passphrase,
+		address: address.getAddressFromPublicKey(keys.publicKey),
+		publicKey: keys.publicKey,
+		privateKey: keys.privateKey,
+		blsPrivateKey,
+		blsPublicKey,
+		blsPoP,
+	};
+};
```

Comment:
This function is not used in this script.

How about we either remove it, or if we really need it, import it from `framework/test/utils/mocks/account.ts` instead?

Then you can also remove bunch of imports, such as `Mnemonic`, `legacy` and `bls`.

Reply:
we cannot use the `test` folder because those are not transpiled to javascript, but since we dont need ill remove